main.js

```diff
@@ -0,0 +1,69 @@
+let dataSection = document.querySelector("#github-data");
+let nameHeader = document.querySelector("#name-header");
+
+fetch("https://api.github.com/users/aaron-1453")
+  .then(function (response) {
+    return response.json();
+  })
+  .then(function (response) {
+    console.log("h2", response);
+    const h2El = document.createElement("h2");
+    h2El.innerText = response.name;
+    dataSection.appendChild(h2El);
+    return response;
+  })
+
+  //   .then(function () {
+  //     const about = document.createElement("h3");
+  //     about.innerText = "About Me";
+  //     return dataSection.appendChild(about);
+  //   })
+
+  .then(function (response) {
+    const basics = document.createElement("div");
+    basics.className = "basics";
+    basics.innerText = "The Basics";
+    dataSection.appendChild(basics);
+    console.log("h3", response);
```

Comment:
A couple of notes on this section starting on line 22. You don't need to chain another `.then` here, because you already have access to the data you need in the first `.then` block starting on line 8.

Second, you have made your job harder than it needs to be here on lines 23-26, because you don't need to create this HTML content in your javascript. You only need to create content here that you need JS data to complete. So, you could create your HTML to have content like this "The Basics" header, and keep your JavaScript streamlined and efficient by only creating the elements you have to.